npm-shrinkwrap.json

```diff
@@ -47,26 +50,43 @@
     "ast-types": {
       "version": "0.8.15"
     },
+    "async-lock": {
+      "version": "1.0.0"
+    },
     "asynckit": {
       "version": "0.4.0"
     },
     "aws-sign2": {
       "version": "0.6.0"
     },
     "aws4": {
-      "version": "1.5.0"
-    },
-    "babel-code-frame": {
-      "version": "6.16.0"
+      "version": "1.6.0"
     },
     "babel-messages": {
-      "version": "6.8.0"
+      "version": "6.23.0",
```

Comment:
I don't thing that updating **all** dependencies here is a good thing to do.

If some new packages are needed by poller script they should be added, if some package needs updating for this feature (raven?) only it should be updated.
Updating all dependencies will affect other parts of BSI and may cause some unexpected changes and bugs (if they are introduced by these dependencies).

Especially if we want to first dry run poller we want to have this merge to have as little impact on BSI webapp as possible. Updating all dependencies is a big impact that we don't really control.